### ISSUE TITLE
Swift auth docs aren't clear

### ISSUE THREAD

**Issue by ipmb** [tsileo/bakthat#41]
It took me a while to figure out how to upload stuff to a swift server. The docs say `access_key: ACCOUNT:USER` which wasn't clear to me. I couldn't get v2 (aka Keystone) auth working. For v1, these are example settings that work for Rackspace CloudFiles:

```
  access_key: <RACKSPACE_USERNAME>
  auth_url: https://auth.api.rackspacecloud.com/v1.0
  auth_version: '1'
  default_destination: swift
  glacier_vault: ''
  region_name: ''
  s3_bucket: bakthat_test
  secret_key: <RACKSPACE_SECRET_KEY>
```

**Comment by tsileo** [comment]
You're right about the docs, it's not easy to get started with Swift, I only tested auth v1 (the Swift backend have been initially created by @yoyama), I'm working on a lot of features for the new bakthat version (mainly incremental backup support).

Maybe you can update the docs for Rackspace CloudFiles user ? If you haven't the time, I'll do it.

@yoyama any thoughts ? Did you get Keystone working ? (and also keep updated with bakthat, I'll push a lot of changes soon).
